code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -991,6 +991,11 @@
 
 	if(stat == DEAD) return
 
+	var/mob/living/carbon/human/H = src
+	if(!H.internal_organs_by_name["diagnostics unit"])
```

Comment:
This is bad, I suggest just reworking this verb to be linked to the organ, like the aut'akh augments are done.